### PR TITLE
T41: Add user model instead of host name

### PR DESCRIPTION
**Pull request by Animansoubi**
add user model and inserting event with new event model

### CODE REVIEW

File: app/src/main/java/com/gathering/android/event/eventdetail/EventDetailFragment.kt [hunk 1]
```diff
@@ -48,7 +48,6 @@ class EventDetailFragment : Fragment() {
                 is EventDetailViewState.ShowEventDetail -> {
                     imageLoader.loadImage(state.event.photoUrl, binding.imgEvent)
                     binding.tvEventTitle.text = state.event.eventName
-                    binding.tvEventHost.text = state.event.hostName
```
Comment (moyarsuccess) [inline app/src/main/java/com/gathering/android/event/eventdetail/EventDetailFragment.kt:51]: It shouldn't be removed

File: app/src/main/java/com/gathering/android/event/model/EventEntity.kt [hunk 2]
```diff
@@ -4,7 +4,7 @@ import java.io.Serializable
 
 data class EventEntity(
     val eventName: String = "",
-    val hostName: String = "",
```
Comment (moyarsuccess) [inline app/src/main/java/com/gathering/android/event/model/EventEntity.kt:7]: the name should be host not user, the user is not clear.

File: app/src/main/java/com/gathering/android/event/myevent/addevent/AddEventBottomSheetFragment.kt [hunk 3]
```diff
@@ -24,15 +23,21 @@ import com.gathering.android.event.KEY_ARGUMENT_SELECTED_IMAGE
...
+import com.google.firebase.auth.ktx.auth
+import com.google.firebase.ktx.Firebase
 import dagger.hilt.android.AndroidEntryPoint
 import java.util.Calendar
 import javax.inject.Inject
 
 @AndroidEntryPoint
 class AddEventBottomSheetFragment : BottomSheetDialogFragment() {
 
+    private lateinit var auth: FirebaseAuth
```
Comment (moyarsuccess) [inline app/src/main/java/com/gathering/android/event/myevent/addevent/AddEventBottomSheetFragment.kt:39]: The view should know anything about Firebase

File: app/src/main/java/com/gathering/android/event/eventdetail/EventDetailFragment.kt (hunk 1 above)
Comment (Animansoubi) [reply app/src/main/java/com/gathering/android/event/eventdetail/EventDetailFragment.kt:51]: solved

**Review by moyarsuccess — APPROVED** [review]

**Pull request merged**